src/ucp/api/ucp_def.h

```diff
@@ -361,7 +361,7 @@ typedef struct ucp_listener_accept_handler {
  *                        @ref UCS_ERR_MESSAGE_TRUNCATED error code is returned.
  *                        Otherwise, an @ref ucs_status_t "error status" is
  *                        returned.
- * @param [in]  info      @ref ucp_tag_recv_info_t "Completion information"
+ * @param [in]  info      @ref ucp_tag_info_t "Completion information"
```

Comment:
Why did this reference change from ucp_tag_recv_info_t to ucp_tag_info_t?

Related to this:
 - the brief description for ucp_tag_recv_info_t currently says "Forward declarations"
 - there is no long description for ucp_tag_recv_info_t which means that the entry in the Typedef section of the documentation has no meaningful descriptive text
Can you please add some appropriate text for ucp_tag_recv_info_t?